adabot/circuitpython_bundle.py

```diff
@@ -373,10 +377,10 @@ def new_release(bundle, bundle_path):
         try:
             fetch_bundle(bundle, bundle_path)
             update_info = update_bundle(bundle_path)
-            if update_info:
-                commit_updates(bundle_path, update_info)
-                push_updates(bundle_path)
-            new_release(bundle, bundle_path)
+            # if update_info:
+            #     commit_updates(bundle_path, update_info)
+            #     push_updates(bundle_path)
+            # new_release(bundle, bundle_path)
```

Comment:
I think this might have been temporary for testing and needs to be restored?
```suggestion
            if update_info:
                commit_updates(bundle_path, update_info)
                push_updates(bundle_path)
            new_release(bundle, bundle_path)
```